src/main/java/Duke/Task.java

```diff
@@ -0,0 +1,90 @@
+package Duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected String taskType;
+
+    public Task(String description, String taskType) {
+        if (taskType.equalsIgnoreCase("event")) {
+            String[] listOfInputs = description.split("/at",2);
+            description = listOfInputs[0] + " (at:" + listOfInputs[1] + ")";
+            this.isDone = false;
+            this.taskType = taskType;
+        }
+        else if (taskType.equalsIgnoreCase("deadline")) {
```

Comment:
You have several cases here. Instead of multiple `else if`, perhaps explicit switch case statements could be, See switch layouts: [https://se-education.org/guides/conventions/java/basic.html#layout](https://se-education.org/guides/conventions/java/basic.html#layout).